src/pages/Profile.jsx

```diff
@@ -0,0 +1,63 @@
+//Profile.js
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Accordion from 'react-bootstrap/Accordion';
+import Card from 'react-bootstrap/Card';
+import ListGroup from 'react-bootstrap/ListGroup';
+import Button from 'react-bootstrap/Button';
+
```

Comment:
When I executed your code, I encountered the following errors:

1. 'Container' is not defined.
2. 'Link' is not defined.

I'm uncertain if these errors stem from version conflicts in your code.

Reply:
Fixed